lib/DriverDisplay/DriverDisplayC.h

```diff
@@ -18,26 +17,38 @@ using namespace std;
 
 extern SemaphoreHandle_t spi_mutex;
 
+// namespace DriverDisplayC {
+// public structures
+enum class INDICATOR { OFF, LEFT, RIGHT, WARN };
+enum class INFO_TYPE { INFO, STATUS, WARN, ERROR };
+enum class CONSTANT_MODE { SPEED, POWER };
+enum class DRIVE_DIRECTION { FORWARD, BACKWARD };
+
 class DriverDisplayC : public abstract_task {
 
 public:
-  // public structures
-  enum class INDICATOR { OFF, LEFT, RIGHT, WARN };
-  enum class INFO_TYPE {
-    INFO,
-    STATUS,
-    WARN,
-    ERROR
-  }; // INFO => ILI9341_WHITE, STATUS => ILI9341_GREEN, WARN => ILI9341_PURPLE,
-     // ERROR => ILI9341_RED
+  // INFO:ILI9341_WHITE, STATUS:ILI9341_GREEN,
+  // WARN.ILI9341_PURPLE, ERROR.ILI9341_RED
+
+  static DriverDisplayC *instance() {
+    if (!_instance) {
+      _instance = new DriverDisplayC();
+    }
```

Comment:
Is this construct something that is commonly used with cpp? 
I would actually prefer to just instantiate everythin in `main.cpp` instead..

Reply:
Did all in your way but DriverDisplayC not --> Issue #84